Orange/tests/test_table.py

```diff
@@ -598,6 +598,18 @@ def test_extend(self):
         with self.assertRaises(ValueError):
             d.extend(x)
 
+    def test_copy(self):
+        t = data.Table(np.zeros((5, 3)), np.arange(5), np.zeros((5, 3)))
+
+        copy = t.copy()
+        self.assertTrue(np.all(t.X == copy.X))
```

Comment:
For direct ndarray comparison, I believe it's better (more elaborate) to use `np.testing.assert_equal`.